recipes/fast-dds/all/conanfile.py

```diff
@@ -147,7 +146,17 @@ def validate(self):
                 raise ConanInvalidConfiguration("Mixing a dll {} library with a static runtime is a bad idea".format(self.name))
 
     def build(self):
-        self._patch_sources()
```

Comment:
would be nice to remove the method as well, it's just dead code now.

Reply:
Agree pushed an update